site/e2e/tests/outdatedCLI.spec.ts

```diff
@@ -11,8 +11,7 @@ import {
 } from "../helpers";
 import { beforeCoderTest } from "../hooks";
 
-// we no longer support versions prior to single tailnet: https://github.com/coder/coder/commit/d7cbdbd9c64ad26821e6b35834c59ecf85dcd9d4
-const clientVersion = "v0.27.0";
+const clientVersion = "v2.8.0";
```

Comment:
Theoretically, this should work with 2.7.0. I tried it and it doesn't work.  Rather than try to debug, I just set it to 2.8.0, which is what it'll be in the PR above this one on the stack.